lib/src/package_context.dart

```diff
@@ -132,6 +136,13 @@ class PackageContext {
     }
   }
 
+  Future<List<ScreenshotResult>> processScreenshots(
```

Comment:
This is a memorized method that takes two parameters... but don't return a new result of the parameters change.

That's probably a bad idea. We could document this behavior in a documentation string, so that people know that this method will forever return results for whatever it was called with first.

Or we could contemplate making it a getter for `screenshotResults` that lazily processes screenshots.
It might be nice to document the laziness in a documentation string as it affects performance characteristics, or maybe it's better to make it a method without parameters.

It's unclear to me how `pkgDir` is different from `PackageContext.packageDir`, similarly why can't we get `screenshots` from `PackageContext.pubspec`.

Reply:
done